http://bugs.python.org/issue24557
assume egd unless OPENSSL_NO_EGD is defined-remove configure check

--- Modules/_ssl.c.orig	2016-06-25 21:52:32 UTC
+++ Modules/_ssl.c
@@ -3325,7 +3325,7 @@ Returns 1 if the OpenSSL PRNG has been s
 It is necessary to seed the PRNG with RAND_add() on some platforms before\n\
 using the ssl() function.");
 
-#ifdef HAVE_RAND_EGD
+#ifndef OPENSSL_NO_EGD
 static PyObject *
 PySSL_RAND_egd(PyObject *self, PyObject *args)
 {
@@ -3753,7 +3753,7 @@ static PyMethodDef PySSL_methods[] = {
      PySSL_RAND_bytes_doc},
     {"RAND_pseudo_bytes",   PySSL_RAND_pseudo_bytes, METH_VARARGS,
      PySSL_RAND_pseudo_bytes_doc},
-#ifdef HAVE_RAND_EGD
+#ifndef OPENSSL_NO_EGD
     {"RAND_egd",            PySSL_RAND_egd, METH_VARARGS,
      PySSL_RAND_egd_doc},
 #endif
